spring-boot-starter-data-aerospike/pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.aerospike</groupId>
+        <artifactId>spring-boot-data-aerospike</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>spring-boot-starter-data-aerospike</artifactId>
+    <name>Spring Boot Data Aerospike Starter</name>
+    <description>Starter for using Spring Data Aerospike</description>
```

Comment:
Shouldn't we have separate reactive starter? How is it done for other storages?

Reply:
My mistake. Need to create reactive started too.
I've done.